src/Orders/OrderIdTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Craftzing\Laravel\MollieWebhooks\Orders;
+
+use Craftzing\Laravel\MollieWebhooks\Exceptions\InvalidResourceId;
+use Craftzing\Laravel\MollieWebhooks\Testing\PrefixedResourceIdTestCase;
+
+final class OrderIdTest extends PrefixedResourceIdTestCase
+{
+    protected function resourceIdClass(): string
+    {
+        return OrderId::class;
+    }
+
+    protected function expectedPrefix(): string
+    {
+        return 'ord_';
+    }
+
+    protected function expectedExceptionClass(): string
```

Comment:
We can remove this one as an abstract method on `PrefixedResourceIdTestCase`. The exception class should be the same for any prefixed resource ID.

Reply:
And make the `prefix` method static?